yarn.lock

```diff
@@ -5023,46 +5028,16 @@ mkpath@^1.0.0:
   resolved "https://registry.yarnpkg.com/mkpath/-/mkpath-1.0.0.tgz#ebb3a977e7af1c683ae6fda12b545a6ba6c5853d"
   integrity sha1-67Opd+evHGg65v2hK1Raa6bFhT0=
 
-mocha-webdriver-runner@^0.5.3:
-  version "0.5.3"
-  resolved "https://registry.yarnpkg.com/mocha-webdriver-runner/-/mocha-webdriver-runner-0.5.3.tgz#ccf496b949631d0b6b5f439e06250f932bf61ad6"
-  integrity sha512-3oXAFCYzlZ+LEK6mI3qKYohIuISmAv5PAg5Fz+C9Tj36CobQglt9M07aip5xh+ImxEz52WZ0nN6fnnMdzveFgw==
+mocha-webdriver-runner@^0.6.2:
+  version "0.6.2"
+  resolved "https://registry.yarnpkg.com/mocha-webdriver-runner/-/mocha-webdriver-runner-0.6.2.tgz#31291065c3c10556c25731ab31ba2c46cb1556af"
+  integrity sha512-f2RjX+iDjJeJaVsFuBjHUN2bfSgAisjHXXvPSLJZLJp5oyONztsc4auaPhbu70omhKD9pkvowZzx8pKXM0Eozg==
   dependencies:
     "@zbigg/treesync" "^0.3.0"
-    commander "^2.20.0"
-    events "^3.0.0"
+    commander "^5.0.0"
+    events "^3.1.0"
     lodash "^4.17.15"
-    mocha "^6.2.0"
-    selenium-webdriver "^4.0.0-alpha.4"
-
-mocha@^6.2.0:
-  version "6.2.2"
-  resolved "https://registry.yarnpkg.com/mocha/-/mocha-6.2.2.tgz#5d8987e28940caf8957a7d7664b910dc5b2fea20"
-  integrity sha512-FgDS9Re79yU1xz5d+C4rv1G7QagNGHZ+iXF81hO8zY35YZZcLEsJVfFolfsqKFWunATEvNzMK0r/CwWd/szO9A==
-  dependencies:
-    ansi-colors "3.2.3"
-    browser-stdout "1.3.1"
-    debug "3.2.6"
-    diff "3.5.0"
-    escape-string-regexp "1.0.5"
-    find-up "3.0.0"
-    glob "7.1.3"
-    growl "1.10.5"
-    he "1.2.0"
-    js-yaml "3.13.1"
-    log-symbols "2.2.0"
-    minimatch "3.0.4"
-    mkdirp "0.5.1"
```

Comment:
This is the main motivation for the change. mkdirp 0.5.1 depends on minimist with security issues, hence github security warning